internal/controller/eventing/controller.go

```diff
@@ -219,12 +218,6 @@ func (r *Reconciler) handlePublisherProxy(
 		return deployment, err
 	}
 
-	// CreateOrUpdate HPA for publisher proxy deployment
```

Comment:
Why is this removed as part of this ticket?

Reply:
because we moved it to the general function [EventingManager.DeployPublisherProxyResources()](https://github.com/muralov/eventing-manager/blob/3de07d9b6e39c30e474aedb539b4a492ebc61f6d/pkg/eventing/manager.go#L166) where it'll be created as other resources